internal/server/scripts.go

```diff
@@ -26,6 +26,9 @@ const (
 	maxLuaPoolSize = 1000
 )
 
+// For Lua os.clock() impl
+var startedAt time.Time
```

Comment:
This can be simplified as `var startedAt = time.Now()`, with the `init()` function removed altogether.

Reply:
added in [02a614b](https://github.com/tidwall/tile38/pull/663/commits/02a614bc6db5102a9cd2d63c92d2349139481b03).